--- ./src/bap_mc.ml     2018-12-11 05:50:02.616110627 +0000
+++ ./src/bap_mc.ml.patched     2018-12-11 05:49:53.664238989 +0000
@@ -19,7 +19,7 @@

   let bad_insn addr state mem start =
     let stop = Addr.(Dis.addr state - addr |> to_int |> ok_exn) in
-    raise (Bad_insn (Dis.memory state, start, stop))
+    Dis.step state (stop + 1)

   let escape_0x =
     String.substr_replace_all ~pattern:"0x" ~with_:"\\x"
